4.6.2/Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:4.6.2
 
-RUN powershell -Command Add-WindowsFeature NET-Framework-45-ASPNET; \
-    powershell -Command Add-WindowsFeature Web-Asp-Net45; \
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
Personally I use PowerShell in all the images I build too, so I understand why the ASP.NET team would want to make this change.

It would be nice, however, if all Windows "base" images had mostly similar defaults, and in this case that would be cmd for the shell...

As @tianon mentioned I understand if you don't want to change this back.